util/cli/cli.go

```diff
@@ -137,14 +137,23 @@ func ReadAndConfirmPassword() (string, error) {
 	}
 }
 
+// SetLogFormat sets a logrus log format
+func SetLogFormat(logFormat string) {
```

Comment:
I think this method should explicitly test for valid log formats. I.e. in the flag, it says:

> Set the logging format. One of: text|json

But actually, you can give it any value.

Reply:
Thanks for your comment! I fixed to output log when the given log format is invalid (just like `errors.CheckError` does in `SetLogLevel` method)!